src/main/java/lux/fm/bookingservice/service/impl/BookingServiceImpl.java

```diff
@@ -87,6 +90,12 @@ public BookingResponseDto updateBookingById(
                                 "Booking with such id doesn't exist: " + id
                         )
                 );
+        if (booking.getPayment().getStatus().equals(Payment.Status.PENDING)) {
+            expireStripeSession(booking);
```

Comment:
In this place we don't need expire

Reply:
<img width="215" alt="Screenshot 2023-11-21 at 18 34 52" src="https://github.com/lux-fm-team/booking-service/assets/61597780/47845a9f-d83f-4eba-9323-fd8e023d9538">